### PR TITLE
Create roadmap.md

### PR DESCRIPTION
**Pull request by radhi1991**
Adding Roadmap file

### CODE REVIEW

File: roadmap.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Adhoc Scan ---> https://github.com/Comcast/xGitGuard/issues/24
+
+| Feature                   | Description                                             | Status | Developer (GitHub ID) |
+|---------------------------|---------------------------------------------------------|--------|-----------------------|
+| Targeted repository scanning | Enable xGitGuard to scan user specified repositories for secrets        | ✅ Done | [preethid03](https://github.com/preethid03) |
+| Targeted organization scanning | Enable xGitGuard to scan user specified organization for secrets        | ✅ Done | [preethid03](https://github.com/preethid03) |
+| Filtering archived repositories | Exclude archived repositories from scanning          | ✅ Done | [preethid03](https://github.com/preethid03) |
+| Filtering forked repositories | Exclude forked repositories from scanning            | ✅ Done | [preethid03](https://github.com/preethid03) |
+| Custom keyword search     | Search for specific keywords within repositories       | ✅ Done | [preethid03](https://github.com/preethid03) |
```
Comment (dinpraka) [inline roadmap.md:22]: 1. This was not completed

**Comment by radhi1991** [comment]
@dinpraka  - Made the requested changes

File: roadmap.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Adhoc Scan ---> https://github.com/Comcast/xGitGuard/issues/24
+
+| Feature                   | Description                                             | Status | Developer (GitHub ID) |
+|---------------------------|---------------------------------------------------------|--------|-----------------------|
+| Targeted repository scanning | Enable xGitGuard to scan user specified repositories for secrets        | ✅ Done | [preethid03](https://github.com/preethid03) |
+| Targeted organization scanning | Enable xGitGuard to scan user specified organization for secrets        | ✅ Done | [preethid03](https://github.com/preethid03) |
+
+## File Scanner 
+
+| Feature                   | Description                                             | Status | Developer (GitHub ID) |
```
Comment (dinpraka) [inline roadmap.md:23]: This should come under separate feature
Filescanner scope is different. Please fix this

**Review by dinpraka — CHANGES_REQUESTED** [review]
1. Roadmap for filescanner is not right. Custom keyword search would be separate feature
2. Add line number feature to roadmap

File: roadmap.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+|---------------------------|---------------------------------------------------------|--------|-----------------------|
+| Training and building models using BERT | Train ML models for secret detection using BERT            | 🚧 To Do | [](https://github.com/developer8) |
+| Integrating BERT into scanners | Integrating trained BERT model into the xGitGuard scanner  | 🚧 To Do | [](https://github.com/developer9) |
+
+## Pre-commit Hook
+
+| Feature                   | Description                                             | Status | Developer (GitHub ID) |
+|---------------------------|---------------------------------------------------------|--------|-----------------------|
+| Multi-language package manager | Manage code checks and linters before commits       |  🚧 To Do | [](https://github.com/preethid030) |
```
[the remark this answers is not in the document]
Comment (radhi1991) [reply roadmap.md]: Done

**Review by dinpraka — APPROVED** [review]

**Pull request merged**